web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -482,7 +482,8 @@ export function UnifiedResources(props: UnifiedResourcesProps) {
             )}
             isProcessing={
               resourcesFetchAttempt.status === 'processing' ||
-              getPinnedResourcesAttempt.status === 'processing'
+              getPinnedResourcesAttempt.status === 'processing' ||
+              unifiedResourcePreferencesAttempt.status === 'processing'
```

Comment:
I don't think it's enough, you'll end up displaying both the resources and the loading skeleton in a situation where the resources have been fetched but we're still to receive prefs.

Reply:
In my local version, I got around this by doing this:

```typescript
            mappedResources={
              // Hide the resources until the preferences are fetched.
              // ViewComponent supports infinite scroll, so it shows both already loaded resources
              // and a loading indicator if needed.
              hasFinished(unifiedResourcePreferencesAttempt)
                ? resources.map(unifiedResource => ({
                    item: mapResourceToViewItem(unifiedResource),
                    key: generateUnifiedResourceKey(unifiedResource.resource),
                  }))
                : []
            }
```

Reply:
Thanks, done.